doctrine/doctrine-bundle/1.6/config/packages/doctrine.yaml

```diff
@@ -1,6 +1,6 @@
 doctrine:
     dbal:
-        url: '%env(DATABASE_URL)%'
+        url: '%env(resolve:DATABASE_URL)%'
```

Comment:
Can you add a comment for people using 3.3? Something like `# If you are using Symfony 3.3, remove the resolve: prefix`